src/widgets/types.ts

```diff
@@ -374,6 +374,7 @@ export type Form = {
   retired: boolean;
   encounterTypeUuid?: string;
   encounterTypeName?: string;
+  checked?: boolean;
```

Comment:
Hmm... I think `checked` isn't really a property a `Form`. It's a characteristic of how we display the form (think "V" in "MVC"). If that's information that the view can infer on its own, then just have the view do so. If it actually expresses something deeper about the Form, e.g. that the form is complete, give it a name that signifies that, like `complete?: boolean;`.